### DISCUSSION TITLE
Change audio output to internal speakers?

### DISCUSSION THREAD

**Discussion by geofftaylor** [ttscoff/bunch-docs#48] [Q&A] [answered]
I feel kind of dumb for not being able to figure this out, but what's the correct `(audio)` syntax to change the output to the internal speakers?

I've tried `(audio output internal)` and `(audio output imac)` to no avail. ("imac" is the computer name in the Sharing preferences.)

`(audio output scarlett)` changes the output to my Scarlett 2i2, but I can't figure out how to change it back to the internal speakers. For context, I'm trying to do this:

```
# Set output to Scarlett when opening the bunch
(audio output scarlett)

# Launch some apps...

# Set output to internal speakers when closing the bunch
!(audio output internal)
```

(I'm on Bunch beta 83 and macOS 11.2.3.)

**Comment by ttscoff** [comment]
You need to use the name of the output as shown by the OS. Option-click on the volume icon in your menu bar and see what your speakers are called. On my MacBook Pro, they're called "MacBook Pro Speakers", and I can switch to them using `(audio output macbook)`. 

I would think the same would be true on iMac, but see what it tells you in the volume dropdown.

**Reply by geofftaylor** [reply]
The name in the volume menu is "Internal Speakers."

<img width="301" alt="Screen Shot 2021-04-09 at 5 26 44 PM" src="https://user-images.githubusercontent.com/518246/114242411-d682cc00-9958-11eb-9fc8-6ac1838f7f83.png">

**Comment by ttscoff** [comment]
That's really odd. I may have to create a build that can dump a list of 
all known audio devices to the log, but before I do that, what happens 
if you try `(audio output speakers)`?

-Brett

**Reply by geofftaylor** [reply]
That didn't work either.

**Comment by ttscoff** [comment]
If you turn on debug level logging, are there any errors listed when running that bunch?

**Reply by geofftaylor** [reply]
No errors, but it looks like maybe the command is getting truncated.

`2021-04-11 11:55:45: ⚪[piano] Processing command "udio output interna"`

In the bunch file, that line is `!(audio output internal)`

**Comment by ttscoff** [comment]
Yeah, that would be the issue :). I'll dig into why that's happening.

**Comment by ttscoff** [comment, marked as answer]
Yep, I had it double trimming the brackets, a casualty of a recent 
refactoring. Will be fixed in the next update.
